lib/sc_client.rb

```diff
@@ -320,13 +320,43 @@ def publish(channel, data)
   # @param [String] data Data to be published on the channel
   # @param [Lambda] ack Block to execute on publish acknowledgment
   #
-  # @return [<type>] <description>
+  #
   #
   def publishack(channel, data, ack)
     @ws.send(get_publish_object(channel, data, increment_cnt).to_json)
     @acks[@cnt] = [channel, ack]
   end
 
+  # initializes logger instance and sets logger level
+  #
+  #
+  #
+  #
+  def initialize_logger
+    @logger = Logger.new(STDOUT)
+    @logger.level = Logger::WARN
+  end
+
+  #
+  # Returns Logger object
+  #
+  #
+  # @return [Object] A logger Object
+  #
+  def get_logger
+    initialize_logger unless @logger
+    return @logger
+  end
+
+  # Disables loging of messages
+  #
+  #
+  #
+  #
+  def disable_logging
```

Comment:
option should be provided to enable or disable logger. Maybe keep method that take bool as a argument, that enable or disables logging...
e.g. scclient.enableLogging(True/False)